### PR TITLE
EG-3732: Fix reference node.conf, correct forward-looking statement, cross-link to node naming docs

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

**Comment by joeldudleyr3** [comment]
@ivanterziev-r3 I've noticed I've failed to reflect the same changes on the identical node.conf pages under the `Corda Operations and Deployment` section. Do you need me to make the changes there as well?

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

File: content/en/docs/corda-os/4.5/corda-configuration-fields.md [hunk 1]
```diff
@@ -315,6 +315,7 @@ Please do not change.
   This acts as a human-readable alias to the node's public key and can be used with the network map to look up the node's info.
   This is the name that is used in the node's certificates (either when requesting them from the doorman, or when auto-generating them in dev mode).
   At runtime, Corda checks whether this name matches the name in the node's certificates.
+  The name must be a valid X.500 distinguished name, as per the [node naming constraints](node-naming.md).
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.5/corda-configuration-fields.md:318]: Should we also change the sentence in line 281: 
Longer term these keys will be managed in secure hardware devices.** 
to
Alternatively, these keys can be managed in secure hardware devices.**

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-fields.md [hunk 2]
```diff
@@ -447,7 +447,7 @@ The New Relic configuration leverages the Dropwizard NewRelicReporter solution.
 The password to unlock the KeyStore file (`<workspace>/certificates/sslkeystore.jks`) containing the node certificate and private key.
 
 **Important: This is the non-secret value for the development certificates automatically generated during the first node run.
-Longer term these keys will be managed in secure hardware devices.**
+Alternatively, these keys can be managed in secure hardware devices.**
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-fields.md:450]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-fields.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-fields.md [hunk 3]
```diff
@@ -490,6 +490,7 @@ The legal identity of the node.
 This acts as a human-readable alias to the node's public key and can be used with the network map to look up the node's info.
 This is the name that is used in the node's certificates (either when requesting them from the doorman, or when auto-generating them in dev mode).
 At runtime, Corda checks whether this name matches the name in the node's certificates.
+The name must be a valid X.500 distinguished name, as per the [node naming constraints](node-naming.md).
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-fields.md:493]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-fields.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 4]
```diff
@@ -165,22 +165,20 @@ Here are the contents of the `reference.conf` file:
         "cordapps"
     ],
     "keyStorePassword" : "MYPASSWORD",
-    "myLegalName" : " MYLEGALNAME",
+    "myLegalName" : "MYLEGALNAME",
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:168]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 5]
```diff
@@ -165,22 +165,20 @@ Here are the contents of the `reference.conf` file:
...
     "detectPublicIp" : false, // Do not perform any public IP lookup on the host.
 
-
     "networkServices" : {
-    "doormanURL" : "https://doorman.uat.corda.network/",
-    "networkMapURL" : "https://netmap.uat.corda.network/"
-},
+        "doormanURL" : "https://doorman.uat.corda.network/",
+        "networkMapURL" : "https://netmap.uat.corda.network/"
+    },
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:178]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 6]
```diff
@@ -189,6 +187,7 @@ Here are the contents of the `reference.conf` file:
         },
         "dataSourceClassName" : "com.microsoft.sqlserver.jdbc.SQLServerDataSource"
     },
+
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:190]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 7]
```diff
@@ -198,6 +197,7 @@ Here are the contents of the `reference.conf` file:
         },
         "dataSourceClassName" : "org.postgresql.ds.PGSimpleDataSource"
     },
+
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:200]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 8]
```diff
@@ -212,22 +212,22 @@ Here are the contents of the `reference.conf` file:
         "schema" : "dbo",
         "transactionIsolationLevel" : "READ_COMMITTED"
     },
+
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:215]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 9]
```diff
@@ -212,22 +212,22 @@ Here are the contents of the `reference.conf` file:
...
-        "password" : "test1",
- "user" : "user1",
-        "permissions" : [ "ALL" ]
-    }
+    "rpcUsers" : [
+        {
+            "password" : "test1",
+            "user" : "user1",
+            "permissions" : [ "ALL" ]
+        }
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:225]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

File: content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md [hunk 10]
```diff
@@ -212,22 +212,22 @@ Here are the contents of the `reference.conf` file:
...
     ],
     "trustStorePassword" : "PASSWORD",
     "sshd" : {
-    "port" : "2222"
-
-},
+        "port" : "2222"
+    },
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/setup/corda-configuration-file.md:230]: Same change needs made in:
`../content/en/docs/corda-enterprise/4.5/operations/deployment/corda-configuration-file.md`

**Review by ivanterziev-r3 — CHANGES_REQUESTED** [review]

**Comment by joeldudleyr3** [comment]
@ivanterziev-r3 Addressed.

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**